### PR TITLE
Reindex last hour txs

### PR DESCRIPTION
**Pull request by Uxio0**
Prevents indexing issues

### CODE REVIEW

**Review by fmrsabino — CHANGES_REQUESTED** [review]
Looks good.

Although I'm trying to understand exactly how this flow would help us: from what I could gather, every 110 minutes we reindex the last 2 hours (default).

While this is ongoing will the indexing of new blocks stop until this task finishes?

**Comment by Uxio0** [comment]
That's correct, but indexing is never stopped. It's run on parallel

File: safe_transaction_service/history/management/commands/setup_service.py [hunk 1]
```diff
@@ -75,6 +75,12 @@ def create_task(self) -> Tuple[PeriodicTask, bool]:
         5,
         IntervalSchedule.MINUTES,
     ),
+    CeleryTaskConfiguration(
+        "safe_transaction_service.history.tasks.reindex_master_copies_last_hours",
```
Comment (fmrsabino) [inline safe_transaction_service/history/management/commands/setup_service.py:79]: Should this be `safe_transaction_service.history.tasks.reindex_last_hours`

**Review by fmrsabino — CHANGES_REQUESTED** [review]

File: safe_transaction_service/history/management/commands/setup_service.py (hunk 1 above)
Comment (Uxio0) [reply safe_transaction_service/history/management/commands/setup_service.py:79]: Sure! Nice catch!

Comment (Uxio0) [reply safe_transaction_service/history/management/commands/setup_service.py]: Should be fixed now

File: safe_transaction_service/history/management/commands/setup_service.py [hunk 2]
```diff
@@ -75,6 +75,12 @@ def create_task(self) -> Tuple[PeriodicTask, bool]:
...
+        "Reindex master copies for the last hours",
+        110,
```
[the remark this answers is not in the document]
Comment (Uxio0) [reply safe_transaction_service/history/management/commands/setup_service.py:81]: Yes

**Review by fmrsabino — APPROVED** [review]

**Pull request merged**